rdr_client/work_queue.py

```diff
@@ -0,0 +1,21 @@
+""" Uses the service account 'awardee-pitt' to get participant summaries for work queue api
+    with an awardee of PITT"""
```

Comment:
You might consider making awardee be flag-configurable. Though for now this is probably fine.